stdlib/public/Observation/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#===--- CMakeLists.txt - Reflection support library ---------------------===#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2023 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+#===----------------------------------------------------------------------===#
+
+add_subdirectory(Sources/_ObservationRuntime)
```

Comment:
@Azoy I didn't want to take such a general name as `_Runtime` like the `Reflection` module. Do we want to change that one too?

Reply:
ah ok, in this case I don't want my runtime module to be public - instead I want it to be an implementation detail of the Observation module

Reply:
Resolved by just reincorporating these types and functions back into a single module